dev_install.sh

```diff
@@ -1,15 +1,21 @@
 #!/bin/bash
-# Ubuntu 14.04 for nodeJS and Ruby development.
+# Ubuntu 14.10 for development.
+
+# Repositories
+sudo add-apt-repository ppa:webupd8team/sublime-text-3
+sudo apt-add-repository ppa:paolorotolo/android-studio
+sudo apt-get update
 
 # Dependencies
-sudo apt-get install python-software-properties curl -y
+sudo apt-get install curl -y
 
 # Main Dev Libraries
 sudo apt-get install build-essential
 
-# Setup Git
-git config --global user.email "ryananthonyrichardson@gmail.com"
-git config --global user.name "Ryan Richardson"
+# Git
+sudo apt-get install git
+git config --global user.email "epwashkow@gmail.com"
+git config --global user.name "EvanWashkow"
```

Comment:
Let's get these 2 lines changed to take command line arguments. Might be a better way, but [this seems easy enough](http://how-to.wikia.com/wiki/How_to_read_command_line_arguments_in_a_bash_script).